events/views.py

```diff
@@ -219,3 +225,22 @@ def stats(request):
 
     return render_to_response(
         'stats/index.html', data, context_instance=RequestContext(request))
+
+def events_category(request, id, slug):
+    """
+        Filter evets for category
+    """
+    try:
+        events = CategoryEvent.objects.filter(category_id=id)
```

Comment:
@francjp do we want to have multiple categories for each event? I think this leads to problems with stats. For example in the Reps portal we have returned back to just one category per event.

My view is one category per event and if we want more filtering we can implement tags latter.